.changeset/big-mayflies-begin.md

```diff
@@ -3,3 +3,4 @@
 ---
 
 Update orchestration to 2501a release.
+As part of this change, the `modelsGet` method of `ScenarioAPI` has been renamed to `scenarioQueryModels`.
```

Comment:
[req] When we draft release, we merge change sets https://github.com/SAP/ai-sdk-js/blob/61954018b53399f76d34a4772ed1ae593a0dd4b4/.github/workflows/draft-release.yml#L12

And we will group changes based on the type: https://github.com/SAP/cloud-sdk-js/blob/e69f2d74edf66eca901da687420f7085466187da/build-packages/merge-changelogs/index.ts#L7

So change set should always start with 

```
  'Known Issue',
  'Compatibility Note',
  'New Functionality',
  'Improvement',
  'Fixed Issue',
  'Updated Dependencies'
```

For example, 

```
[Compatibility Note] Update ...

Reply:
I mean this is technically a breaking change 😆

Reply:
Resolved.